pyquiz/expr/matrix.py

```diff
@@ -533,15 +533,15 @@ def scale(i, c):
             mat[i][k] *= c
         if steps_out != None:
             Ri = var(f"R_{i+1}")
-            steps_out.append(rf"{c * Ri} \rightarrow {Ri}")
+            steps_out.append(rf"{Ri} \rightarrow {c * Ri}")
     def replace(i, j, c):
         # R_i + c * R_j -> R_i
```

Comment:
```suggestion
        # R_i ~~> R_i + c * R_j
```
The other comments for the other functions should get similar treatment (`R_i <~~> R_j` for swap and `R_i ~~> c * R_i` for scale).